Controllers/ExploreController.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Net.Mime;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+
+namespace Explorer.Controllers
+{
+    [ApiController]
+    [Produces(MediaTypeNames.Application.Json)]
+    public class ExploreController : ControllerBase
+    {
+        private readonly ILogger<ExploreController> _logger;
+
+        public ExploreController(ILogger<ExploreController> logger)
+        {
+            _logger = logger;
+        }
+
+        [HttpPost]
+        [Route("explore")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        public IActionResult Explore(Models.ExploreParams data)
+        {
+            if (!ModelState.IsValid)
+            {
+                return BadRequest(ModelState);
```

Comment:
The errors produced are not very descriptive. Or rather they are descriptive but the problem is that the field names that are being complained about as missing do not match the field names as they would be provided in the input JSON.

Here is an example from the tests you provided in slack:

```
- wrong data in request JSON (missing api_key)
> curl -k -X POST -H "Content-Type: application/json" https://localhost:5001/explore  -d '{"table_name":"t1", "column_name":"c2"}'
{"type":"https://tools.ietf.org/html/rfc7231#section-6.5.1","title":"One or more validation errors occurred.","status":400,"traceId":"|ecdc1210-4b8aaa2689f85747.","errors":{"ApiKey":["The ApiKey field is required."]}}
```

Notice that the request should have contained `api_key` but your error complains about `ApiKey`.

Reply:
I did some improvements on that, but it is still not ideal. I will try to search some other solutions to fix this completely.

Reply:
Before we go too far down this route: If there is no simple obvious solution to this problem, then I am totally happy with changing the parameter names in the API instead! I.e. have the payload be: 

```json
{
  "ApiKey": "...",
  "TableName": "...",
  "ColumnName": "..."
}
```

instead.

That way the problem kind of solved itself?